browser/static/js/adjust.js

```diff
@@ -304,6 +304,10 @@ class ImageAdjuster {
       singleOutline = true;
       o1 = brush.value;
     }
+    if (editMode && currentHighlight && !brush.conv) {
+      singleOutline = true;
+      o1 = brush.value;
+    }
 
     this.postCompositeLabelMod(
```

Comment:
Can you just add a comment here about what this is doing? The conditional is not obvious to me.